src/App.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import Navbar from './components/navbar/Navbar';
 import Home from './pages/home/Home';
-import Calculations from './pages/calculations/Calculations';
+import Calculations from './components/calculator/Calculator';
 import Quotes from './pages/quotes/Quotes';
 import Footer from './components/footer/Footer';
 
```

Comment:
- [ ] As mentioned in the project requirements please make sure we **Use class based components (All components should be class-based).** currently our App component is a functional component you can convert it to a class component 
```
class App extends React.PureComponent {
  render() {
    return (
      <>
        <BrowserRouter>
          <Navbar />
          <Routes>
            <Route path="/" element={<Home />} />
            <Route path="/calculator" element={<Calculator />} />
            <Route path="/quotes" element={<Quotes />} />
          </Routes>
          <Footer />
        </BrowserRouter>
      </>
    );
  }
}
```